services_app/src/main/java/org/opendatakit/services/utilities/ODKServicesPropertyUtils.java

```diff
@@ -38,6 +38,9 @@ public static void clearActiveUser(PropertiesSingleton props) {
       properties.put(CommonToolProperties.KEY_USERNAME, "");
       properties.put(CommonToolProperties.KEY_PASSWORD, "");
 
+      properties.put(CommonToolProperties.KEY_IS_USER_AUTHENTICATED,"false");
```

Comment:
Is it possible to keep it as actual boolean value instead of wrapping boolean value in a string?

Reply:
No, the **Properties** Class only stores the values as a String, so we cannot keep it as an actual boolean value.
Reference - https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html